webapp/src/components/Bid/Bid.js

```diff
@@ -135,20 +136,20 @@ export default class Bid extends React.PureComponent {
     } = this.props
 
     const hasSameSellerAndBidder = bid.seller === bid.bidder
+    const fingerprintChanged = fingerprintHasChanged(bid)
     const assetDetailLink = this.getDetailLink()
     const sizeByResolution = {
       computer: showAssetDetail ? 4 : 3,
       tablet: showAssetDetail ? 4 : 3,
       mobile: showAssetDetail ? 8 : 16
     }
 
+    let gridClassName = `Bid ${className}`
+    gridClassName += showAssetDetail ? ' showAssetDetail' : ''
+    gridClassName += fingerprintChanged ? ' fingerprint-changed' : ''
```

Comment:
Maybe this could be a method?
also, to keep css classes consistent maybe `showAssetDetail` should be `show-asset-detail` ?